Sources/XCRemoteCache/Commands/Postbuild/PostbuildContext.swift

```diff
@@ -83,6 +83,8 @@ public struct PostbuildContext {
     /// location of the json file that define virtual files system overlay
     /// (mappings of the virtual location file -> local file path)
     let overlayHeadersPath: URL
+    /// Regexes of files that should not be included in the dependency list
+    let skippedDependenciesRegexes: [String]
```

Comment:
What do you think about renaming this property to match the `config`'s one `irrelevantDependenciesPaths`? I think it would make the code easier to follow. Up to you.

Reply:
Sure, renamed.